src/PollController.js

```diff
@@ -78,14 +105,15 @@ function (Okta, FormController) {
       },
 
       initialize: function (options) {
-        this.transaction = options.appState.get('transaction');
-        this.factorPollingInterval = this.transaction.factor.profile.refresh;
-        const factorPollingIntervalSeconds = Math.floor(this.factorPollingInterval/1000);
-        this.title = Okta.loc('polling.title','login', [factorPollingIntervalSeconds]);
+        this.transactionObject = options.appState.get('transaction');
+        this.factorPollingIntervalSeconds = Math.ceil(this.transactionObject.transaction.profile.refresh/1000);
+        this._updateTitle(this.factorPollingIntervalSeconds);
       },
   
       postRender: function () {
-        this._startPolling(this.factorPollingInterval);
+        setTimeout(() => {
+          this._startCountDown(this.factorPollingIntervalSeconds);
```

Comment:
I guess we dont need the setTimeout here as the startCountDown takes care of calling doPoll after the factorPollingIntervalSeconds anyway.